input/rmatch.in

```diff
@@ -1,94 +1,94 @@
+    qscan_flag=f         ! Scan over eigenvalue Q; not used with RPEC
+    sol_flag =f          ! Newton solve for single eigenvalue; not used with RPEC
+    deflate=f            ! Divides Delta determinant by PRODUCT(guess-oldroots)
+    nroot=5              ! Number of eigenvalues to find
+    itermax=5000         ! Maximum number of iterations in Newton solve
+    relax_fac=0.5        ! Numerical relaxation factor in Newton solve
+    msing=4              ! Number of singular surfaces to include in matching
+    initguess = ( 0,10)  ! Initial guess for eigenvalues in Newton solve
+    scan_x0 = 1.90E-10   ! Start of scan range if doing any scan
+    scan_x1 =  1e-9      ! End of scan range if doing any scan
+    scan_nstep = 50      ! Number of steps in scan if doing any scan
+    qscan_ising=2        ! Which singular surface to scan Q over
+    deltar_flag=t        ! Run and record Galerkin GGJ model in Fourier space
+    deltac_flag=t        ! Run and record Galerkin GGJ model in configuration space
+    deltaj_flag=t        ! Run and record Jardin finite element GGJ model
+    qscan_out=f          ! Output delta from Q scan?
 
-    scan_nstep = 50
+    ! Resistive MHD options
+    eta=8e-8,8e-8,8e-8,8e-8   ! Resistivity at each rational surface, starting from the core
+    massden=3.3e-07,3.3e-07,3.3e-07,3.3e-07   ! Mass density (kg/m^3) at each rational surface, starting from the core. 3.3e-7 is ni=1e20 m^-3.
+    model="deltac"       ! Which method to use for computing inner layer response. Options are "deltar", "deltac", "deltaj". 
 
-    model="deltac"
-    qscan_ising=2
-    deltar_flag=t
-    deltac_flag=t
-    deltaj_flag=t
-    qscan_out=f
-
-    match_sol%flag=t
-    match_sol%uniform=t
-    match_sol%qpert=2
-    match_sol%auto_connect=f
-    match_sol%connect_threshold=1e-3
-    match_sol%b_flag=f
-    coil%ideal_flag=f
+    match_sol%flag=t     ! Write outsol_qpert.bin
+    match_sol%uniform=t  ! Use a uniform grid in psi?
+    match_sol%qpert=2    ! Which mode to write in outsol_qpert.bin?
+    match_sol%auto_connect=f ! run match.f/match_auto_connect; false in examples, unknown precise behavior
+    match_sol%connect_threshold=1e-3 ! Proportional difference between outer and inner solutions for match_auto_connect to connect them
+    match_sol%b_flag=f   ! Write solutions in terms of b instead of xi
+    coil%ideal_flag=f    ! Write ideal galerkin solution instead of resistive -- combined with GPEC this is physically equivalent to running regular DCON
 /
 &RMATCH_OUTPUT
-    bin_rpecsol=f
-    out_rpecsol=f
+    bin_rpecsol=f        ! Write outer and inner solutions to binary files
+    out_rpecsol=f        ! Write outer and inner solutions to ascii files
 /
 &DELTAC_LIST
-    tid=3,5,6,4
-    deltac_bin_sol=f
-    deltac_out_sol=f
+    tid=3,5,6,4          ! Something to do with which solution indexes are used in computing combined solutions. Always 3,5,6,4 in examples.
```

Comment:
You might already know but I'll share some context: the inner region equations evaluated in deltac have 6 components (see Eq. 2 in Glasser and Wang 2020 https://doi.org/10.1063/1.5134999 ). tid refers to specific components of the inner region equations such as perturbed normal mag field, perturbed normal plasma displacement etc (why he chooses 3,5,6,4 I'm have no clue)